spinalcordtoolbox/scripts/sct_dmri_moco.py

```diff
@@ -201,6 +201,7 @@ def main(argv: Sequence[str]):
     param.remove_temp_files = arguments.r
     if arguments.param is not None:
         param.update(arguments.param)
+    param.verbose = verbose
```

Comment:
Just out of curiosity, in the issue, you mentioned:

> `verbose=2` is changed for `verbose=1` here:
> https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/aa4aa7b038745e86b3058c5f8819de999d5c01ed/spinalcordtoolbox/moco.py#L753

Is this unrelated? (In other words, the *true* issue is that the verbose argument from the script wasn't properly being passed to the moco API function?)

Reply:
> Did this turn out to be unrelated? (In other words, the true issue is that the verbose argument from the script wasn't properly being passed to the param object in the moco function?)

indeed-- unrelated, and the issue is that verbose was not passed